src/DependencyInjection/Configuration.php

```diff
@@ -36,7 +44,25 @@ public function getConfigTreeBuilder()
                 ->scalarNode('revision_field_name')->defaultValue('rev')->end()
                 ->scalarNode('revision_type_field_name')->defaultValue('revtype')->end()
                 ->scalarNode('revision_table_name')->defaultValue('revisions')->end()
-                ->scalarNode('revision_id_field_type')->defaultValue('integer')->end()
+                ->scalarNode('revision_id_field_type')
+                    ->defaultValue('integer')
+                    // NEXT_MAJOR: Use validate() instead.
+                    ->beforeNormalization()
+                        ->always(static function ($value) {
+                            if (null !== $value && !\in_array($value, self::ALLOWED_REVISION_ID_FIELD_TYPE, true)) {
+                                @trigger_error(sprintf(
+                                    'The value "%s" for the "revision_id_field_type" is deprecated'
+                                    .' since sonata-project/entity-audit-bundle 1.3 and will throw an error in version 2.0.'
```

Comment:
```suggestion
                                    .' since sonata-project/entity-audit-bundle 1.x and will throw an error in version 2.0.'
```

Reply:
I'm the only PR opened, and since it's a deprecation, it will be a minor. I win some time ^^'